Pod/Classes/PINRemoteImageManager.m

```diff
@@ -189,7 +188,7 @@ - (instancetype)initWithSessionConfiguration:(NSURLSessionConfiguration *)config
         if (!configuration) {
             configuration = [NSURLSessionConfiguration ephemeralSessionConfiguration];
         }
-        _callbackQueue = dispatch_queue_create("PINRemoteImageManagerCallbackQueue", DISPATCH_QUEUE_CONCURRENT);
+        _callbackQueue = dispatch_get_main_queue();
```

Comment:
Since this queue can be changed externally at any time, we will either need to protect the getter / setters of this properties with a PINRemoteLock instance for thread-safety. Another alternative is to create another initializer where the caller can specify the queue and remove the property from the public interface entirely.